src/models/PostsModel.php

```diff
@@ -36,4 +37,23 @@ public function getSingle(int $id): PostEntity|false
         return $query->fetch();
     }
 
+    public function addComment(CommentEntity $commentEntity): bool
+    {
+        $query = $this->db->prepare(
+            'INSERT INTO `comments` (`content`,`username_id`, `post_id`, `date_posted`, `time_posted`) VALUES (:content, :username_id, :post_id, :date_posted, :time_posted);');
+        return $query->execute([':content' => $commentEntity->content, ':username_id' => $commentEntity->username_id, ':post_id' => $commentEntity->post_id, ':date_posted' => $commentEntity->date_posted, ':time_posted' => $commentEntity->time_posted]);
+    }
```

Comment:
Is it appropriate for the PostsModel to add comments to the comments table? Nothing in this method is directly to do with posts.